x-pack/plugins/infra/public/components/autocomplete_field/autocomplete_field.tsx

```diff
@@ -307,6 +307,7 @@ const SuggestionsPanel = euiStyled(EuiPanel).attrs({
   position: absolute;
   width: 100%;
   margin-top: 2px;
-  overflow: hidden;
+  overflow: scroll;
```

Comment:
This leads to horizontal scrollbars in most cases if the browser window is a bit smaller. How about leaving `overflow-x` on `hidden` and only scrolling on `overflow-y`?

```suggestion
  overflow: hidden scroll;
```

![grafik](https://user-images.githubusercontent.com/973741/55872271-dd6d7d00-5b8c-11e9-8cb9-d5b4913c0e88.png)